drizzlepac/haputils/product.py

```diff
@@ -283,79 +332,132 @@ def align_to_gaia(self, catalog_list=[], output=True, process_type='SVM',
+                                    )
+                                )
+                                align_table.imglist = align_table.perform_fit(
+                                    method_name,
+                                    catalog_item,
+                                    ref_catalog,
+                                    fitgeom=mosaic_fitgeom,
+                                )
 
                                 # Define comparison for min cross-matches based on fitgeom used
-                                alignment_pars['determine_fit_quality']['min_xmatches'] = \
-                                    alignment_pars['run_align']['mosaic_fitgeom_list'][mosaic_fitgeom]
+                                alignment_pars["determine_fit_quality"][
+                                    "min_xmatches"
+                                ] = alignment_pars["run_align"]["mosaic_fitgeom_list"][
+                                    mosaic_fitgeom
+                                ]
 
                                 # turn off consistency check for SVM and MVM processing since filter-to-filter
                                 # or visit-to-visit offsets could be large relative to measurement RMS.
-                                alignment_pars['determine_fit_quality']['do_consistency_check'] = False
+                                alignment_pars["determine_fit_quality"][
+                                    "do_consistency_check"
+                                ] = False
 
                                 # Evaluate the quality of the fit
                                 # Need to create this log file specifically for the PyTest testing environment
                                 # when a single Python session is running and the tests are looking for dataset-specific
                                 # log files.
                                 log_file = "temp_align.log"
-                                is_good_fit, _, _, _, _, _ = align.determine_fit_quality_mvm_interface(align_table.imglist,
-                                                                                                       align_table.filtered_table,
-                                                                                                       more_catalogs,
-                                                                                                       num_cat,
-                                                                                                       alignment_pars,
-                                                                                                       print_fit_parameters=True,
-                                                                                                       loglevel=self.log_level,
-                                                                                                       runfile=log_file)
-                                
+                                (
```

Comment:
Please just revert the text for Lines 399-406 to the unBlack state which is easier to read (IMHO).